build.xml

```diff
@@ -710,9 +710,9 @@
           <dependency groupId="com.google.j2objc" artifactId="j2objc-annotations" version="1.3"/>
           <!-- adding this dependency is necessary for assertj. When updating assertj, need to also update the version of
              this that the new assertj's `assertj-parent-pom` depends on. -->
-          <dependency groupId="org.junit" artifactId="junit-bom" version="5.6.0" type="pom" scope="test"/>
```

Comment:
I would be more comfortable with bumping this in a separate ticket rather than burying it in a patch like this, for better visibility. Is is also absolutely necessary to bump that or is this mostly an opportunistic "why not" bump?

Reply:
`SoftAssertions` is the reason. If it was a separate ticket it would be "why not bump" 😄

Reply:
sorry - you asking specifically about "bom" or assertj?

Reply:
@jacek-lewandowski basically both ... sp whats the reason for bom?

Reply:
I guess you just followed this, correct?

    <!-- adding this dependency is necessary for assertj. When updating assertj, need to also update the version of
             this that the new assertj's `assertj-parent-pom` depends on. -->

Reply:
yes

Reply:
ok, so I think it is all justified.